static/src/javascripts/projects/commercial/modules/header-bidding/utils.ts

```diff
@@ -142,11 +142,11 @@ export const shouldIncludeXaxis = (): boolean => isInUk();
 
 export const shouldIncludeImproveDigital = (): boolean => isInUk() || isInRow();
 export const shouldIncludeImproveDigitalSkin = (): boolean => {
-	if (!(isInUk() || isInRow())) return false;
 	return (
+		isInVariantSynchronous(improveSkins, 'variant') &&
```

Comment:
@mxdvl need to measure both control and variant

Reply:
This ensures only the variant gets the placement id. Commercial metrics are sent for all variants.

Reply:
Got it, my mistake. Go for it then